src/LodeRunner.Core/Models/LoadTestConfig.cs

```diff
@@ -147,5 +146,28 @@ public class LoadTestConfig : BaseEntityModel
         /// </value>
         [Description("--dry-run")]
         public virtual bool DryRun { get; set; }
+
+        /// <summary>
+        /// Sets mock LoadTestConfig data.
+        /// </summary>
+        /// <param name="name">LoadTestConfig name.</param>
+        public void SetMockData(string name)
```

Comment:
Move mock data generation.

Reply:
Addressed in commit by adding content to files in LodeRunner.API.Test/IntegrationTests/Extensions